content/concurrency.md

```diff
@@ -1,14 +1,57 @@
 ## VIII. Concurrency
 ### Scale out via the process model
 
-Any computer program, once run, is represented by one or more processes.  Web apps have taken a variety of process-execution forms.  For example, PHP processes run as child processes of Apache, started on demand as needed by request volume.  Java processes take the opposite approach, with the JVM providing one massive uberprocess that reserves a large block of system resources (CPU and memory) on startup, with concurrency managed internally via threads.  In both cases, the running process(es) are only minimally visible to the developers of the app.
+#### 1\. A twelve-factor app treats processes as first-class citizens.
 
-![Scale is expressed as running processes, workload diversity is expressed as process types.](/images/process-types.png)
+Any computer program, once run, is represented by one or more processes.
+In the twelve-factor app, processes take strong cues from
+[the unix process model for running service daemons](https://adam.herokuapp.com/past/2011/5/9/applying_the_unix_process_model_to_web_apps/).
+Using this model, the developer can architect their app to handle diverse
+workloads by assigning each type of work to a *process type*. For example,
```

Comment:
The example seemed like it should be broken out into examples, but it wasn't really landing well with the others. I think we are missing one more key concept so I broke them out as, 1/ Any computer program, once run, is represented by one or more processes, 2/ A twelve-factor app treats processes as first-class citizens, 3/ A twelve-factor app scales out by assigning workloads to separate process types, and 4/ A twelve-factor app relies on the execution environment to manage processes.

Reply:
That’s funny I think it initially broke them out that way but the 1st concept didn’t have much explanatory text. I can go back to that version if it feels more natural to you